parsec/backend/organization.py

```diff
@@ -140,10 +142,20 @@ async def api_authenticated_organization_stats(self, client_ctx, msg):
             }
         )
 
-    @api("organization_bootstrap", client_types=[ClientType.APIV1_ANONYMOUS])
+    @api("organization_bootstrap", client_types=[ClientType.APIV1_ANONYMOUS, ClientType.ANONYMOUS])
     @catch_protocol_errors
-    async def api_organization_bootstrap(self, client_ctx, msg):
-        msg = apiv1_organization_bootstrap_serializer.req_load(msg)
+    async def api_organization_bootstrap(
+        self, client_ctx: Union[APIV1_AnonymousClientContext, AnonymousClientContext], msg: dict
+    ):
+        # Use the correct serializer depending on the API
+        if isinstance(client_ctx, APIV1_AnonymousClientContext):
+            serializer = apiv1_organization_bootstrap_serializer
+        else:
+            assert isinstance(client_ctx, AnonymousClientContext)
+            serializer = organization_bootstrap_serializer
+
+        msg = serializer.req_load(msg)
+
         bootstrap_token = msg["bootstrap_token"]
```

Comment:
should add a comment above where `if "redacted_user_certificate" in msg:` is done given this won't be needed once APIv1 is dropped